File: unused/Edge.py
from Node import Node

class Edge:
    def __init__(self, node_1: Node, node_2: Node):
        if node_1.node_id == node_2.node_id:
            raise ValueError("Cannot contain self-edge")

        self.node1_id = node_1.node_id
        self.node2_id = node_2.node_id
        self.edge_id = str(min(self.node1_id, self.node2_id)) + "-" + str(max(self.node1_id, self.node2_id))

File: unused/Edge.R
setClass("Edge", 
         slots = c(
           Node1_id = "numeric",
           Node2_id = "numeric",
           Node1_disease_status = "character",
           Node2_disease_status = "character"
         )
) -> Edge

setMethod("initialize", "Edge", function(.Object, Node1, Node2) {
  .Object@Node1_id = Node1@id
  .Object@Node2_id = Node2@id
  .Object@Node1_disease_status = Node1@disease_status
  .Object@Node2_disease_status = Node2@disease_status
  return(.Object)
})
